### ISSUE TITLE
running tests with `python2 runtests.py` doesn't work

### ISSUE THREAD

**Issue by duk3luk3** [FAForever/client#454]
Maybe a linux problem?

See here:
* https://github.com/FAForever/client/pull/446#issuecomment-258618619

`py.test` somewhat works:

```
(fafclient) luke@Max client(linux_support_2↑3|✔) % py.test                                                                                                                                                                    ~/git/faf/client
============================================================================================================= test session starts =============================================================================================================
platform linux2 -- Python 2.7.12, pytest-3.0.3, py-1.4.31, pluggy-0.4.0
PyQt4 4.11.4 -- Qt runtime 4.8.7 -- Qt compiled 4.8.7
rootdir: /home/luke/git/faf/client, inifile: 
plugins: qt-2.1.0, mock-1.4.0, cov-2.4.0
collected 31 items 

tests/fa/test_featured.py ............
tests/fa/test_maps.py .
tests/fa/test_updater.py ............
tests/unit_tests/client/test_player.py ..
tests/unit_tests/client/test_updating.py .s
tests/unit_tests/games/test_gameitem.py ..

=========================================================================================================== pytest-warning summary ============================================================================================================
WC1 /home/luke/git/faf/client/tests/fa/test_featured.py cannot collect test class 'TestObjectWithoutIsFinished' because it has a __init__ constructor
WC1 /home/luke/git/faf/client/tests/fa/test_featured.py cannot collect test class 'TestThreadNoOp' because it has a __init__ constructor
=========================================================================================== 30 passed, 1 skipped, 2 pytest-warnings in 0.66 seconds ===========================================================================================
```

But not with `pyqt4v2`:
```
(fafclient) luke@Max client(linux_support_2↑3|✔) % export PYTEST_QT_API='pyqt4v2'                                                                                                                                             ~/git/faf/client
                                                                                
(fafclient) luke@Max client(linux_support_2↑3|✔) % py.test                                                                                                                                                                    ~/git/faf/client
INTERNALERROR> Traceback (most recent call last):         
INTERNALERROR>   File "/home/luke/scratch/venvs/fafclient/lib/python2.7/site-packages/_pytest/main.py", line 92, in wrap_session
INTERNALERROR>     config._do_configure()
INTERNALERROR>   File "/home/luke/scratch/venvs/fafclient/lib/python2.7/site-packages/_pytest/config.py", line 859, in _do_configure
INTERNALERROR>     self.hook.pytest_configure.call_historic(kwargs=dict(config=self))
INTERNALERROR>   File "/home/luke/scratch/venvs/fafclient/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 750, in call_historic
INTERNALERROR>     self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
INTERNALERROR>   File "/home/luke/scratch/venvs/fafclient/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 339, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR>   File "/home/luke/scratch/venvs/fafclient/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 334, in <lambda>
INTERNALERROR>     _MultiCall(methods, kwargs, hook.spec_opts).execute()
INTERNALERROR>   File "/home/luke/scratch/venvs/fafclient/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 614, in execute
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/home/luke/scratch/venvs/fafclient/lib/python2.7/site-packages/pytestqt/plugin.py", line 167, in pytest_configure
INTERNALERROR>     qt_api.set_qt_api(config.getini('qt_api'))
INTERNALERROR>   File "/home/luke/scratch/venvs/fafclient/lib/python2.7/site-packages/pytestqt/qt_compat.py", line 75, in set_qt_api
INTERNALERROR>     sip.setapi("QDate", 2)
INTERNALERROR> ValueError: API 'QDate' has already been set to version 1
```

**Comment by duk3luk3** [comment]
Build uses pyqtv1: https://github.com/FAForever/client/commit/2283c420e6b73dc4034a121a58b3ea90f47b877b

Needs fixed too.

**Comment by duk3luk3** [comment]
Fixed in ee687a9

**Comment by duk3luk3** [comment]
I screwed something up merging #457. Needs to be looked at and fixed.
